packages/ui/components/StickyHeader.tsx

```diff
@@ -0,0 +1,49 @@
+import classnames from 'classnames'
+import { PropsWithChildren } from 'react'
+
+export interface StickyHeaderProps
+  extends React.DetailedHTMLProps<
+      React.HTMLAttributes<HTMLElement>,
+      HTMLElement
+    >,
+    PropsWithChildren {
+  variant?: 'primary' | 'light' | 'dark'
+  size?: 'sm' | 'md' | 'lg'
```

Comment:
i think not needed. we can just have the height be dynamic so it can adjust based on the content/children height.